f2/src/forms/PrivacyConsentInfoForm.js

```diff
@@ -1,109 +1,64 @@
-      {false ? ( // mark ids for lingui
-        <div>
-          <Trans id="privacyConsentInfoForm.yes" />
-          <Trans id="privacyConsentInfoForm.consent" />
-        </div>
-      ) : null}
-
-      <Form
-        initialValues={whetherConsent}
-        onSubmit={(values) => {
-          props.onSubmit(values)
-        }}
-        validate={validate}
-        render={({
-          handleSubmit,
-          values,
-          errors,
-          submitFailed,
-          hasValidationErrors,
-        }) => (
-          <Stack
-            as="form"
-            onSubmit={handleSubmit}
-            shouldWrapChildren
-            spacing={6}
-          >
-            {submitFailed && hasValidationErrors ? (
-              <ErrorSummary>
-                <Trans id="privacyConsentInfoForm.hasValidationErrors" />
-              </ErrorSummary>
-            ) : null}
-
-            <FormArrayControl
-              name="consentOptions"
-              errorMessage={<Trans id="privacyConsentInfoForm.warning" />}
-            >
-              {consentOptions.map((key) => {
-                return (
-                  <React.Fragment key={key}>
-                    <CheckboxAdapter name="consentOptions" value={key}>
-                      <Trans id="privacyConsentInfoForm.yes" />
-                      <A
-                        href={'/privacystatement?lang=' + i18n.locale}
-                        isExternal
-                      >
-                        <Trans id="privacyConsentInfoForm.linkOut" />
-                      </A>
-                      <Trans id="privacyConsentInfoForm.period" />
-                    </CheckboxAdapter>
-                  </React.Fragment>
-                )
-              })}
-            </FormArrayControl>
-            <NextAndCancelButtons
-              next={
-                fyiForm ? (
-                  <Trans id="fyiForm.nextPage1" />
-                ) : (
-                  <Trans id="privacyConsentInfoForm.nextPage" />
-                )
-              }
-              button={<Trans id="privacyConsentInfoForm.nextButton" />}
-            />
-          </Stack>
-        )}
-      />
-    </React.Fragment>
+    <Formik
+      initialValues={whetherConsent}
+      onSubmit={(values) => {
+        props.onSubmit(values)
+      }}
+      validationSchema={PrivacyConsentInfoFormSchema()}
+    >
+      {({ handleSubmit, handleChange, handleBlur }) => (
+        <Form onSubmit={handleSubmit}>
+          <Container>
+            <Row className="form-question">
+              <ErrorMessage name="consentOptions" component={Error} />
+            </Row>
+            <Row className="form-section">
+              <Field
+                name="consentOptions"
+                label={
+                  <Trans id="privacyConsentInfoForm.yes.withExternalLink">
+                    <A
+                      color="#0000ff"
+                      target="_blank"
+                      href={'/privacystatement?lang=en'}
```

Comment:
This link defaults to the English privacy statement. The link should use the language option the user selected.